downloader/src/main/kotlin/Main.kt

```diff
@@ -237,7 +254,7 @@ object Main {
      *
      * @throws DownloadException In case the download failed.
      */
-    fun download(target: Package, outputDirectory: File): File {
+    fun download(target: Package, outputDirectory: File): DownloadResult {
```

Comment:
Please also adjust the docs for the return type above.

Reply:
Done.